Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhart/alpine-node:base
```

Comment:
It is best practice to use the official node base image. Who knows when/if this base image will get security patches.

I would suggest using `FROM node:8.11-alpine`